diophantine.tex

```diff
@@ -0,0 +1,55 @@
+The word "Diophantine" comes with the name Diophantus, a mathematician lived
+in the great city of Alexandria sometime around 250 AD. Often referred to as
+the "father of Algebra", Diophantus in his famous work "Arithmetica"
+presented 150 problems that marked the early beginnings of number theory, the
+field of study about integers and their properties. Diophantine equations play
+a central and an important part in number theory.
```

Comment:
It's more than is needed here. I don't think we need to explain the history of the equations, etc.